Onboarding/Onboarding/StepDefinitions/CertificationStepDefinitions.cs

```diff
@@ -11,105 +12,109 @@ namespace Onboarding.StepDefinitions
     [Binding]
     public class CertificationStepDefinitions : CommonDriver
     {
-        [After]
-        public void Dispose()
-        {
-            if (driver != null)
-            {
-                driver.Close();
-            }
-        }
+        //Define Pages and Objects
+        Certification CertificationObj;
+        LoginPage LoginPageObj;
 
-        Certification CertificationObj = new Certification();
         [Given(@"I signed into the portal")]
         public void GivenISignedIntoThePortal()
         {
             //Open Chrome browser
             driver = new ChromeDriver();
+            CertificationObj = new Certification(driver);
```

Comment:
here you can create a constructor  like so

public CertificationStepDefinitions{
            CertificationObj = new Certification(driver);

}